components/lib/crash/src/main/java/mozilla/components/lib/crash/CrashReporter.kt

```diff
@@ -96,6 +96,13 @@ class CrashReporter(
         return this
     }
 
+    /**
+     * Get a copy of the crashBreadCrumbs
+     */
+    fun crashBreadcrumbsCopy(): ArrayList<Breadcrumb> {
+        return ArrayList<Breadcrumb>(crashBreadcrumbs)
+    }
```

Comment:
This shouldn't be a public method for everyone to call :)

(Afaik you can use `.clone()` for that)

Reply:
My bad.  Should have been internal.  I'll update.  Thanks,

Reply:
Updated to use clone as well 👍